### PR TITLE
Fix inappropriate term in "benefits" section

### PR DESCRIPTION
**Pull request by emceeaich**
The word "crazy" is a slur and not keeping with our community participation guidelines. https://www.mozilla.org/about/governance/policies/participation/

## Summary

Proposed change:

Closes # .

## Checklist

I've completed the checklist below to ensure I didn't forget anything. This makes reviewing this PR as easy as possible for the maintainers. And it gets this change released as soon as possible.

* [ ] Proposed change helps towards [*purpose of this project*](https://github.com/mozilla/nunjucks/blob/master/CONTRIBUTING.md#purpose).
* [ ] [*Documentation*](https://github.com/mozilla/nunjucks/tree/master/docs/) is added / updated to describe proposed change.
* [ ] [*Tests*](https://github.com/mozilla/nunjucks/tree/master/tests) are added / updated to cover proposed change.
* [ ] [*Changelog*](https://github.com/mozilla/nunjucks/blob/master/CHANGELOG.md) has an entry for proposed change (if user-facing fix or feature).

### CODE REVIEW

**Comment by fdintino** [comment]
I will assume this pull request was opened in good faith, so I am happy to merge it. But it should be noted that "crazy" in the sense of "crazy extensible" is a north-americanism that simply means "extremely extensible" or "very extensible" and does not have even a remote relationship to mental illness. To be honest I would prefer changing it to say "Very extensible with with custom filters and extensions," both to retain the original meaning and also because it is a more similar construction to the other bullet points.

**Comment by emceeaich** [comment]
I'm happy with "Very" as an intensifier here. Let me update the PR. Thanks!

**Comment by Spown** [comment]
170 open tickets, 25 other prs, but no, this sjw nonsense is the only thing that matters.

I'm actually against this pr on the grounds of not caving in under pressure by people who aggressively seek to censor and thought police the Internet and the real life, by misusing the rules meant to be used in good faith. The word "crazy" is only then a slur when applied to a person and this particular case is not such an occasion. Don't get me wrong, the world would be a better place if no one would try to use derogative terms against other people, but this requires the elimination of this intention, not eradication of all "bad" words from the language (you can always invent more).

If the word "crazy" shifts into meaning "wild", "uncontrollable", "extensible", etc (which it was doing in the course of the last decades), slowly loosing it's applicability to living things, via organic process of language change - it will only be a good thing. When you try to interdict in this process with "cease & desist" you'll only make things worse. Not only will you remind about the word's insulting option (i.e. prevent it from fading), but you'll make it into an inciting taboo.

In short: no one was called crazy and nobody was trying to insult anybody. This is an appropriate usage of this word. A hundred thousand of such appropriate uses will make this word loose it's ability to be a slur. By inhibiting this process with censure you'll only achieve the opposite.

Please stop doing it.

**Comment by fdintino** [comment]
@Spown I agree with some of what you wrote, but I'm still inclined to merge this pull request. I'd like to share my reasoning.

Given that this pull request exists, at least one person found the language objectionable. I think this is worth keeping in mind, and all other things being equal I think that's reason enough to merge. It doesn't cost me anything to do it. And, in fact, this change is an improvement for other reasons. A non-native English speaker reading these docs might not be familiar with this colloquial meaning of 'crazy.' In documentation, I think plain language is generally better.

That said, I myself have a serious mental illness, and I don't personally have an issue with the original language. I mention this because it's relevant here, but I have real anxiety about a current or future employer reading that admission. There is still a very real stigma around mental illness, and by comparison I tend to regard issues around language and usage as trivial. I think you're right that 'crazy extensible' here is pretty far removed from actual mental illness—it's intended to be read as an intensifier, akin to 'wildly,' and it actually carries a positive connotation here. But words can mean different things to different people.

If I might make a suggestion to @emceeaich, and I hope this isn't impertinent, I think that describing the wording as a 'slur' and 'inappropriate' isn't particularly helpful. I didn't write this documentation, but I can imagine that whoever did might take these characterizations personally—the implication being that the writer used an inappropriate slur, when I think it's safe to say that there was no such intention. To avoid knee-jerk defensiveness, I think it would be better to find more neutral ways of proposing changes such as this in the future.

**Comment by fdintino** [comment]
@emceeaich can you check the box on this pull request to 'Allow edits from maintainers'? That way, I can resolve the conflict in the changelog for the merge.
